src/seedu/addressbook/AddressBook.java

```diff
@@ -138,13 +138,28 @@
 
 
     /* We use a String array to store details of a single person.
-     * The constants given below are the indexes for the different data elements of a person
+     * The constants in the enum given below are the indexes for the different data elements of a person
      * used by the internal String[] storage format.
      * For example, a person's name is stored as the 0th element in the array.
      */
-    private static final int PERSON_DATA_INDEX_NAME = 0;
-    private static final int PERSON_DATA_INDEX_PHONE = 1;
-    private static final int PERSON_DATA_INDEX_EMAIL = 2;
+    private enum PersonIndices {
+        PERSON_DATA_INDEX_NAME(0),
```

Comment:
Probably could have just used ordinal, but then if the order of the enums were ever changed that would cause problems.